requirements.txt

```diff
@@ -0,0 +1 @@
+pygame==2.6.0
```

Comment:
**Suggestion:** Add a newline at the end of the `requirements.txt` file.

**Existing Code:**
```Text
pygame==2.6.0
```

**Improved Code:**
```Text
pygame==2.6.0
```

**Details:**
Add a newline at the end of the `requirements.txt` file to adhere to best practices.